src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/Tile.scala

```diff
@@ -28,13 +30,18 @@ class Tile(var tileId: TileId, var latitudeNorth: Double, var latitudeSouth: Dou
   def midWestLongitude(): Double = (centerLongitude() + longitudeWest) / 2.0
 }
 
+object TileUtils {
+  def apply(): TileUtils = new TileUtils
+
+}
 /**
   * Created by erisch on 5/24/2017.
   */
 
 class TileUtils {
```

Comment:
I wonder if we should just implement all these tile-utils in Java and add them to [timpark/geotile](https://github.com/timfpark/geotile) so that others can re-use this. Happy to take that if you want.